CHANGELOG.md

```diff
@@ -198,8 +205,28 @@ Backwards compatible changes
 
 * The contents of `Data.Covec` is now polymorphic with respect to levels
 
+* Added new proofs to `Data.Fin.Properties`:
+  ```agda
+  ≤-irrelevance : ∀ {n} → IrrelevantRel (_≤_ {n})
+  <-irrelevance : ∀ {n} → IrrelevantRel (_<_ {n})
+  ```
+
+* Added new proofs to `Data.Integer.Properties`:
+  ```agda
+  ≤-irrelevance : IrrelevantRel _≤_
+  <-irrelevance : IrrelevantRel _<_
+  ```
+
+* Added new proofs to `Data.List.All.Properties`:
+  ```agda
+  All-irrelevant : IrrelevantPred P → IrrelevantPred (All P)
```

Comment:
Shouldn't this be named `All-irrelevance` like the one in `Data.Vec.All.Properties`

Reply:
Yes, it should be. Nice spot! Will fix.